src/include/reconcile.i

```diff
@@ -43,23 +43,23 @@ __wt_rec_need_split(WT_RECONCILE *r, size_t len)
  *     Initialize an address timestamp triplet.
  */
 static inline void
-__wt_rec_addr_ts_init(WT_RECONCILE *r, wt_timestamp_t *newest_durable_ts,
-  wt_timestamp_t *oldest_start_tsp, uint64_t *oldest_start_txnp, wt_timestamp_t *newest_stop_tsp,
-  uint64_t *newest_stop_txnp)
+__wt_rec_addr_ts_init(WT_RECONCILE *r, wt_timestamp_t *start_durable_ts,
+  wt_timestamp_t *oldest_start_tsp, uint64_t *oldest_start_txnp, wt_timestamp_t *stop_durable_ts,
+  wt_timestamp_t *newest_stop_tsp, uint64_t *newest_stop_txnp)
 {
     /*
      * If the page is not fixed-length column-store, where we don't maintain timestamps at all, set
      * the oldest/newest timestamps to values at the end of their expected range so they're
      * corrected as we process key/value items. Otherwise, set the oldest/newest timestamps to
      * simple durability.
      */
-    *newest_durable_ts = WT_TS_NONE;
+    *start_durable_ts = WT_TS_NONE;
     *oldest_start_tsp = WT_TS_MAX;
     *oldest_start_txnp = WT_TXN_MAX;
+    *stop_durable_ts = WT_TS_NONE;
```

Comment:
Here's the opposite, where `start_durable_ts` has a "max" of `WT_TS_NONE`, which isn't what I'd expect.

Reply:
As per the discussion in the ticket WT-5668, as both start and stop durable timestamps will hold the maximum value of timestamp on the page, so it is better to use the same initial value for both of them for easier understanding. And we followed the same notation irrespective of the value or address cell.